scripts/src/java/org/oppia/android/scripts/proto/BUILD.bazel

```diff
@@ -1,26 +1,33 @@
+"""
+This library contains all protos used in the scripts module.
+In Bazel, proto files are built using the proto_library() and java_lite_proto_library() rules.
+The proto_library() rule creates a proto file library to be used in multiple languages.
+The java_lite_proto_library() rule takes in a proto_library target and generates java code.
```

Comment:
Suggest also referencing the documentation in model/BUILD.bazel since that provides a lot of context for how to add new proto libraries.

Reply:
Done